packages/core/useEventBus/index.ts

```diff
@@ -45,12 +54,22 @@ export function useEventBus<T = unknown>(key: EventBusIdentifer<T>): UseEventBus
     return _off
   }
 
+  function once(listener: EventBusListener<T>) {
+    function _listener(...args: any[]) {
+      off(listener)
+      // @ts-ignore
+      listener(...args)
+    }
+    _listener._ = listener
+    return on(_listener)
```

Comment:
Interesting solution! But honestly it feels a bit magic, how about:

```suggestion
    on(_listener)
    return () => off(listener)
```

And let's say we don't support to use `bus.off(...)` to remove listeners registered using `once` (which I believe is a very rare use case) but use the stop handler returned by `once`

Reply:
Yes, it is indeed a very interesting implementation, I saw when I checked the information.
There may indeed be this problem here, because when off, only the original method passed in will be compared.
This modification can avoid this problem.